backend/src/files/files.service.ts

```diff
@@ -25,7 +26,7 @@ export class FilesService {
 		private configService: ConfigService,
 		private prismaService: PrismaService
 	) {
-		this.s3Client = new S3Client();
+		this.s3Client = new S3Client(this.getStorageConfig());
```

Comment:
I think it’s time to separate the object creation. This object should be injected from the file module. By doing so, we won’t need to handle the S3 configuration in the service layer.

Please refer to `backend/src/langchain/langchain.module.ts`.